index.js

```diff
@@ -51,49 +55,106 @@ const selectors = {
 };
 
 class Scholar {
-  init(key) {
-    if (!key) return this;
-    this.isProxy = true;
-    client.defaults.options = got.mergeOptions(client.defaults.options, {
+  init(apiKey, { retries } = {}) {
+    const useProxy = !apiKey;
```

Comment:
```suggestion
    const useProxy = !!apiKey;
```
Does `useProxy` need to be `true` if apiKey is set?

Reply:
Absolutely, this is a rather stupid mistake :facepalm: